scripts/ristiinvertailu/Makefile

```diff
@@ -0,0 +1,83 @@
+SHELL := /bin/bash
+DATE_NOW := $(shell date '+%Y-%m-%d')
+ALKUPVM := 2024-11-13 # TODO: Change this!
+OUTPUTS_DIR := outputs
+
+ifndef AWS_PROFILE
+$(error AWS_PROFILE is not set)
+endif
+
+ifndef DDB_JAKSOTUNNUS_TABLE_NAME
+$(error DDB_JAKSOTUNNUS_TABLE_NAME is not set)
+endif
+
+ifndef DDB_AMIS_TABLE_NAME
+$(error DDB_AMIS_TABLE_NAME is not set)
+endif
+
+
+ifeq ($(AWS_PROFILE), oph-prod)
+	ENV_NAME := sade
+else
+	ENV_NAME := pallero
+endif
+
+# POSTGRES
+DB_NAME := ehoks
+DB_USERNAME := app
+DB_PASSWORD != aws ssm get-parameter --profile $(AWS_PROFILE) \
```

Comment:
uuuuu! :+1:

Reply:
Ilmeisesti SSH-tunneli pitää sentään pystyttää ite? :)

Reply:
Dokumentaatioksi: se tehää näin:
```
ssh -F ~/.opintopolku/sade.ssh.config -L 55432:ehoks.db.opintopolku.fi:5432 bastion.sade

Reply:
Aa joo, tää pitää tehdä ite, tuonkin ois tietyisti voinu laittaa osaksi Makefilea :sweat_smile: